packages/contracts-bridge/contracts/mocks/MockAccountant.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+import {NFTRecoveryAccountant} from "../accountants/NFTAccountant.sol";
+
+contract MockNftAccountant is NFTRecoveryAccountant {
+    mapping(address => bool) public allowList;
+
+    constructor() NFTRecoveryAccountant(address(0), address(1)) {}
+
+    function exposed_increaseTotalAffected(address _asset) public {
+        if (totalAffected[_asset] == 0) {
+            totalAffected[_asset] = 5000 ether;
+        } else {
+            totalAffected[_asset] = totalAffected[_asset] + 10 ether;
+        }
+    }
+
+    function exposed_record(
+        address _asset,
+        address _user,
+        uint256 _amount
+    ) external {
+        _record(_asset, _user, _amount);
+    }
+
+    modifier onlyAllowed() {
+        require(allowList[msg.sender], "not allowed");
+        _;
+    }
+
+    function recover(uint256 _id) external onlyAllowed {
+        address _user = ownerOf(_id);
+        require(_user == msg.sender, "only NFT holder can recover");
+        Record memory _rec = records[_id];
+        uint256 _amount = 50000000;
```

Comment:
nit: this is in wei, right? so 0.00000000005 ETH - since this is for the purpose of developing the frontend, preferable to use a more readable value like `0.05 ether`